collectors/python.d.plugin/python.d.conf

```diff
@@ -72,7 +72,7 @@ logind: no
 nginx_log: no
 # nsd: yes
 # ntpd: yes
-openldap: no
+openldap: yes
```

Comment:
`openldap: yes` and `# openldap: yes` is two different things for `python.d.plugin`, please add `#`

Reply:
👍